packages/gatsby-plugin-cms/src/node-api/types.ts

```diff
@@ -0,0 +1,33 @@
+export interface Block {
+  name: string
+  props: Record<string, unknown>
+}
+
+export interface RemotePageContent {
+  remoteId: string
+  name: string
+  type: string
+  builderId: 'faststore'
+  author: {
+    id: string
+    email: string
+  }
+  lastUpdatedAt: string
+  blocks: Block[]
+  beforeBlocks: Block[]
+  afterBlocks: Block[]
+  extraBlocks: Array<{
+    name: string
+    blocks: Block[]
+  }>
+}
+
+export type PageContent = {
```

Comment:
Why not use Interface?

Reply:
Because the `interface` construct does not allow to declare named props alongside record like objects

Reply:
Yeah, i forgot this